VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```
@@ -307,7 +307,14 @@ final public class AuthenticationController
             }
         }
         
-        self.client.authenticatedAccount = nil
+        if loadClientCredentials
+        {
+            self.client.authenticatedAccount = (try? self.accountStore.loadAccount(.ClientCredentials)) ?? nil
```

The bogus values I mean, assuming failure from corrupted data